handlers/BES_Modules_The_HDF5_Handler.adoc

```diff
@@ -130,10 +466,9 @@ data model_
 
 |32-bit signed integer |Int32 |
 
-|64-bit unsigned integer |N/A |Results in an error unless
-H5.IgnoreUnknownTypes is true. See above.
+|64-bit unsigned integer |N/A |DAP2 does not support 64-bit integer type.
```

Comment:
I think N/A is better than "Not Supported".  "Not Supported", in my understanding, is that our software doesn't support it although , in theory, it can be supported. "N/A" means that this cannot be done regardless the software practice. The reason that 64-bit unsigned integer cannot be mapped to DAP2 is because DAP2 doesn't support 64-bit integer, it is not due to the difficulties of software implementation.

Reply:
You make my point for me:
> is because DAP2 doesn't support

Reply:
I will consult James on this.